solr/Dockerfile

```diff
@@ -1,5 +1,6 @@
 #ddev-generated
-FROM solr:9.6
+ARG SOLR_BASE_IMAGE="scratch"
```

Comment:
What is `scratch`?

Reply:
https://hub.docker.com/_/scratch

It's used here to "fix" this warning on build https://docs.docker.com/reference/build-checks/invalid-default-arg-in-from/

We could have the real image name here, but using `scratch` allows us to maintain it only in one place, `docker-compose.solr.yaml`.

Reply:
Thank you!

Reply:
More info: https://hub.docker.com/_/scratch